grpc/pom.xml

```diff
@@ -119,9 +119,9 @@
                 <artifactId>protobuf-maven-plugin</artifactId>
                 <version>0.6.1</version>
                 <configuration>
-                    <protocArtifact>com.google.protobuf:protoc:3.12.0:exe:${os.detected.classifier}</protocArtifact>
+                    <protocArtifact>com.google.protobuf:protoc:3.20.1:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
this fixes the error we saw related to M1 macs - M1 versions don't exist for protoc 3.12.0 and protoc-gen-grpc-java 1.31.1, but they do for later versions